STUDIO/org.openl.rules.webstudio/src/org/openl/rules/webstudio/web/admin/security/InheritedAuthenticationSettings.java

```diff
@@ -0,0 +1,111 @@
+package org.openl.rules.webstudio.web.admin.security;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import io.swagger.v3.oas.annotations.Parameter;
+
+import org.openl.config.PropertiesHolder;
+import org.openl.rules.webstudio.web.admin.SettingPropertyName;
+import org.openl.util.CollectionUtils;
+import org.openl.util.StringUtils;
+
+public class InheritedAuthenticationSettings extends AuthenticationSettings {
+
+    private static final String SEP = ",";
+
+    private static final String ADMINISTRATORS = "security.administrators";
+    private static final String DEFAULT_GROUP = "security.default-group";
+    private static final String ALLOW_PROJECT_CREATE_DELETE = "security.allow-project-create-delete";
+
+    @Valid
+    @NotNull
+    private DBSettings db;
```

Comment:
Move `DBSettings` outside of the security because of the DB is base configuration for whole OpenL Studio